rest_framework-stubs/permissions.pyi

```diff
@@ -1,16 +1,17 @@
-from typing import Any, Dict, List, Protocol, Sequence, Type, Union
+from typing import Dict, List, Protocol, Sequence, Type, Union
 
 from django.db.models import Model, QuerySet
-from django.views import View
 from rest_framework.request import Request
+from rest_framework.views import APIView
 
 SAFE_METHODS: Sequence[str] = ("GET", "HEAD", "OPTIONS")
 
 class _SupportsHasPermission(Protocol):
-    def has_permission(self, request: Request, view: View) -> bool: ...
-    def has_object_permission(self, request: Request, view: View, obj: Any) -> bool: ...
+    def has_permission(self, request: Request, view: APIView) -> bool: ...
+    def has_object_permission(self, request: Request, view: APIView, obj: Model) -> bool: ...
 
 _PermissionClass = Union[Type[BasePermission], OperandHolder, SingleOperandHolder]
+_OperationClass = Union[AND, OR, NOT]
```

Comment:
I think it a rare case but someone could define its own operators

Reply:
Its possible, but than he or she will need to override the typing in only one place - I dont think we should use Any in such circumstances.

Reply:
I'm not sure is it possible in mypy but something like _OperationClass = Type[_SupportsHasPermision] would be optimal

Reply:
its possible, i can define this as a protocol probably.

Reply:
silly me, it was already a protocol. Done.